### ISSUE TITLE
to conquer Oasis

### ISSUE THREAD

**Issue by Sonikboy** [iSparkas/Dboor-Updated-V1.0#2]
Warning: mysqli_fetch_array() expects parameter 1 to be mysqli_result, boolean given in C:\xampp\htdocs\lib\mysql.php on line 24

Warning: mysqli_free_result() expects parameter 1 to be mysqli_result, boolean given in C:\xampp\htdocs\lib\mysql.php on line 33

Notice: Undefined index: artefacts in C:\xampp\htdocs\app\model\battles\war.php on line 519

**Comment by iSparkas** [comment]
Yes, i'm aware of this, i'll have to disable hero mansion till hero fix.
